### PR TITLE
Jordan/rename ci jobs

### PR DESCRIPTION
**Pull request by jbibla**
i have no idea how any of this stuff actually works so this is just an experiment 🙃

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f368c9fc614e9cd1b8d9790--lunieio.netlify.app

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Publish to master
+name: Publish Jobs
```
Comment (Bitcoinera) [inline .github/workflows/publish.yml:1]: I actually find "Publish to master" more eloquent / clear

File: app/.codecov.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-coverage:
-  status:
-    project: no
-    patch: yes
-    changes: no
+# coverage:
+#   status:
+#     project: no
+#     patch: yes
+#     changes: no
```
Comment (Bitcoinera) [inline app/.codecov.yml:9]: Good bye codecov :wave:

File: .github/workflows/publish.yml (hunk 1 above)
Comment (jbibla) [reply .github/workflows/publish.yml:1]: it's the "Publishing Workflow" - not a specific job

Comment (jbibla) [inline .github/workflows/publish.yml:1]:
```suggestion
name: Publishing Workflow
```

File: .github/workflows/app_test.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: App testing
+name: App Jobs
```
Comment (jbibla) [inline .github/workflows/app_test.yml:1]:
```suggestion
name: App Testing Workflow
```

File: .github/workflows/app_deploy.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-name: Publish on Netlify
+name: Deploy on Netlify
```
Comment (jbibla) [inline .github/workflows/app_deploy.yml:1]:
```suggestion
name: App Deployment Workflow
```

File: .github/workflows/api_test.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-name: API testing
+name: API Jobs
```
Comment (jbibla) [inline .github/workflows/api_test.yml:1]:
```suggestion
name: API Testing Workflow
```

**Comment by faboweb** [comment]
should we really ignore testing?

**Comment by jbibla** [comment]
> should we really ignore testing?

what do you mean?

File: .github/workflows/app_deploy_production.yml [hunk 6]
```diff
@@ -1,4 +1,4 @@
-name: Publish on Netlify
+name: App Production Deployment Workflow
```
Comment (Bitcoinera) [inline .github/workflows/app_deploy_production.yml:1]: Nice!

File: app/package.json [hunk 7]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "vue-cli-service build --no-unsafe-inline --skip-plugins pwa --mode production",
     "build:dev": "vue-cli-service build --no-unsafe-inline --skip-plugins pwa --mode development",
-    "buildsentry": "vue-cli-service build --no-unsafe-inline --skip-plugins pwa --modern --mode production && chmod 755 tasks/sentry.sh && ./tasks/sentry.sh",
+    "build:sentry": "vue-cli-service build --no-unsafe-inline --skip-plugins pwa --modern --mode production && chmod 755 tasks/sentry.sh && ./tasks/sentry.sh",
```
Comment (Bitcoinera) [inline app/package.json:9]: Nice

**Comment by Bitcoinera** [comment]
Please, resolve the conflicts, but must say I really like the renaming. Adds much more clarity to the whole

**Comment by faboweb** [comment]
needs a merge. please remove any change that is not renaming relevant:
- readd codecov
- readd polkadot e2e

**Comment by jbibla** [comment]
want to confirm 

- i added yarn --fix in a couple places - will this work as expected?
- i deleted some workflow folders from within the specific projects - they were unused, right?

**Comment by Bitcoinera** [comment]
> * i added yarn --fix in a couple places - will this work as expected?

You mean `yarn lint --fix`? Yes, it should work

> * i deleted some workflow folders from within the specific projects - they were unused, right?

Yeah, they weren't used anymore... I believe, actually. @faboweb ?

**Pull request merged**